### PR TITLE
Features/114

### PR DESCRIPTION
**Pull request by frantsao**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

Added proxy buffer vars in nginx.conf template

### Benefits

More flexibility in configurations

### Possible Drawbacks

N/A

### Applicable Issues

#114

### CODE REVIEW

**Review by santi-eidu — APPROVED** [review]

**Review by xtianae7 — APPROVED** [review]

**Review by sorobon — APPROVED** [review]

File: defaults/main.yml [hunk 1]
```diff
@@ -92,7 +100,7 @@ nginx_compile_time_options:
   - { option: "with-ld-opt", value: "-Wl,-rpath,/usr/local/lib" }  # Disable this option if metrics are disabled
 
 # LuaJIT
-luaJIT_version: '2.1-20200102'
+luajit_version: '2.1-20200102'
```
Comment (ablancolopez) [inline defaults/main.yml]: while we are at it... should we add the role namespace to this variable? `nginx_luajit_version`

File: templates/nginx/nginx.conf.j2 [hunk 2]
```diff
@@ -28,9 +28,18 @@ http {
     keepalive_timeout {{ nginx_keepalive_timeout }};
     keepalive_requests {{ nginx_keepalive_requests }};
     reset_timedout_connection on;
-
-    proxy_buffering on;
-    proxy_buffers 8 4k;
+{% if nginx_proxy_buffering  is defined %}
```
Comment (ablancolopez) [inline templates/nginx/nginx.conf.j2:31]: Aren't this variables always defined in `defaults/main.yml`?

**Review by ablancolopez — APPROVED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (frantsao) [reply defaults/main.yml]: Good point!

File: templates/nginx/nginx.conf.j2 (hunk 2 above)
Comment (frantsao) [reply templates/nginx/nginx.conf.j2:31]: Yes, in the beginning these variables weren't going to be defined in defaults, it was a late decision.

Comment (ablancolopez) [reply templates/nginx/nginx.conf.j2:31]: then, this checks can be removed from the template? :-)

**Pull request merged**